Vagrantfile

```diff
@@ -2,54 +2,15 @@
 # vi: set ft=ruby :
 #
 
-ansible_groups = {
-  "dev:children" => ["dev-mail", "dev-test"],
-  "dev-mail:children" => ["mta", "mda", "db" ],
-  "dev-test" => ["tester"],
-  "mta" => ["north", "south"],
-  "mda" => ["north", "south"],
-  "db" => ["north"],
-}
-
-ansible_host_vars = {
-  "north" => {"ispmail_dovecot_replication_peer" => "south.mail.local"},
-  "south" => {"ispmail_dovecot_replication_peer" => "north.mail.local"}
-}
-
 Vagrant.configure(2) do |config|
-  config.vm.box = "debian/stretch64"
-
-  config.vm.define "north" do |subconfig|
-    subconfig.vm.hostname = "mailserver-north"
-    subconfig.vm.provider "virtualbox" do |vb|
-      subconfig.vm.network "private_network", ip: "10.0.0.100"
-    end
-  end
-  config.vm.define "south" do |subconfig|
-    subconfig.vm.hostname = "mailserver-south"
-    subconfig.vm.provider "virtualbox" do |vb|
-      subconfig.vm.network "private_network", ip: "10.0.0.101"
-    end
-    subconfig.vm.provision "ansible" do |ansible|
-      ansible.playbook = "test/mailserver-vagrant.yml"
-      ansible.limit = "dev-mail"
-      ansible.groups = ansible_groups
-      ansible.host_vars = ansible_host_vars 
-      ansible.become = true
-    end
-  end
-
+  config.vm.box = "debian/buster64"
 
-  config.vm.define "tester" do |subconfig|
-    subconfig.vm.hostname = "tester"
-    subconfig.vm.provision "ansible" do |ansible|
-      ansible.limit = "all"
-      ansible.playbook = "test/tester-vagrant.yml"
-      ansible.become = true
-      ansible.groups = ansible_groups
-    end
+  config.vm.define "docker" do |subconfig|
+    subconfig.vm.hostname = "docker-vm"
     subconfig.vm.provider "virtualbox" do |vb|
-      subconfig.vm.network "private_network", ip: "10.0.0.50"
+      vb.memory=8192
```

Comment:
Whoa, do we really need that much ram/cpus ?

Reply:
Not really, tests are suceeding on 2gio with 2 full mailservers + 1 db + 1 client doing lots of calls.

On my local setup I don't like to wait

Reply:
Hmmm ok for CPUs.
RAM however, if almost none of it ends up used, I'd scale it down.

(yes, that's bikeshedding)

Reply:
updated